src/main/java/edu/psu/swe/commons/jaxrs/utilities/JsonSerializationUtil.java

```diff
@@ -37,16 +38,16 @@
     return Optional.of(obj);
   }
   
-  public static <T> Optional<List<T>> deserializeList(String json, Class<T> clazz) throws IOException {
+  public static <T> List<T> deserializeList(String json, Class<T> clazz) throws IOException {
```

Comment:
It would be nice to mark this method with the @NonNull JSR-305 annotation to help static analysis tools.  Lately I've been favoring ``ErrorProne`` and ``SpotBugs``.

Reply:
I have added this annotation to the method